hal/daler_dc_motor_ctrl.h

```diff
@@ -0,0 +1,58 @@
+/** 
+ * \page The MAV'RIC license
+ *
+ * The MAV'RIC Framework
+ *
+ * Copyright © 2011-2014
+ *
+ * Laboratory of Intelligent Systems, EPFL
+ */
+ 
+
+/**
+ * \file dc_motor_ctrl.h

```

Comment:
It is very specific to the board ludovic made to control his flaps.
I would put it in the folder drivers/ as it does not depend directly from the hardware.